habitat-lab/habitat/config/benchmark/multi_agent/hssd_spot_human_social_nav.yaml

```diff
@@ -0,0 +1,182 @@
+# @package _global_
```

Comment:
Thanks! I prefer the config file being in the same place so that we can easily debug issues. I find this in practice helps me a lot when training policies and debugging issues quickly. For this reason, I suggest that we make social nav config yaml a standalone file.

Reply:
Thanks! I prefer the config file being in the same place so that we can easily debug issues. I find this in practice helps me a lot when training policies and debugging issues quickly. For this reason, I suggest that make social nav config yaml a standalone file. I will add more comments there.